dc/helifuzzy.dc

```diff
@@ -131,7 +132,7 @@ layer helifuzzy is
 	     ||p.t_registered := w.day
 		 ||p.t_last_activity := w.day
 	     ||p.m:=1.0
-		 --||p.goal:= Goal( task -> control_weight(), target := Real3(v1:=71.0, v2:=0.0, v3:=0.0), w:= Real3(v1:=1.0, v2:=0.0, v3:=0))
+		 --||p.goal:= Goal( task -> control_weight(), target := RealVector(v1:=71.0, v2:=0.0, v3:=0.0), w:= RealVector(v1:=1.0, v2:=0.0, v3:=0))
          ||p.status -> hapa_motivation(); 		 

```

Comment:
Do you want to change the goal during execution?

Reply:
No, there is no need to change the goal during execution. This line is commented out